src/sections/footer.html

```diff
@@ -1,4 +1,30 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<!-- Parent -->
+<div class="bg-webGrey px-5 lg:px-9 pt-20 pb-9">
+  <!-- Image   -->
+  <div class="flex justify-center w-full">
+    <img src="/images/red-shape.svg" alt="" class="md:w-[50%] w-[100%]" />
+  </div>
+  <div
+    class="flex md:flex-row flex-col lg:gap-0 gap-6 justify-between md:flex-wrap lg:flex-nowrap pt-20 lg:pt-36 font-medium text-base md:text-lg"
+  >
+    <div class="grow">
+      <p>Agentur Baumeister</p>
+      <p>Böckhstraße 13</p>
+      <p>10967 Berlin</p>
+    </div>
+    <div class="grow md:text-right lg:text-left">
+      <a href="" class="underline">hallo@agentur-baumeister.com</a>
```

Comment:
avoid empty attributes
```suggestion
      <a href="#" class="underline">hallo@agentur-baumeister.com</a>
```